src/lib/y2storage/filesystems/btrfs.rb

```diff
@@ -127,6 +127,35 @@ class Btrfs < BlkFilesystem
       #   @raise [Storage::Exception] if the device cannot be removed
       storage_forward :remove_device
 
+      # @!method btrfs_qgroups
+      #   Collection of Btrfs qgroups of the filesystem
+      #   @return [Array<BtrfsQgroup>]
+      storage_forward :btrfs_qgroups, as: "BtrfsQgroup"
+
+      # @!method has_quota
+      #   Whether quota support is enabled for this btrfs filesystem
+      #
+      #   @return [Boolean]
+      storage_forward :has_quota
```

Comment:
What about making `#has_quota` private?

Reply:
`#has_quota` is a pretty stable method, since it comes from libstorage-ng. I don't see any problem in offering it (it's maybe convenient for people used to the libstorage-ng API). Its only "problem" is that the name is not as rubist as the alias, but I'm not sure whether it deserves to be hidden just for that. :wink: